pkg/registry/apis/query/testdata/legacy-ds-uid.json

```
@@ -0,0 +1,82 @@
+{
+  "description": "datasource uid is variable",
+  "input": {
+    "from": "now-6",
+    "to": "now",
+    "queries": [
+      {
+        "refId": "C",
+        "queryType": "",
+        "datasourceUid": "__expr__",
+        "type": "reduce",
+        "reducer": "last",
+        "datasourceId": 100,
+        "datasource": {
+          "type": "__expr__",
+          "uid": "__expr__"
+        },
+        "expression": "$B"
+      },
+      {
+        "refId": "A",
+        "queryType": "",
+        "datasourceUid": "XYX",
```

`XYX`? Could you put something slightly more meaningful in here? Dunno, maybe something like:

```suggestion
        "datasourceUid": "test_case_datasource_UID",
```

Or even:
```suggestion
        "datasourceUid": "foo",
```